### DISCUSSION TITLE
blog/blog-is-a-toy

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#12] [Announcements]
# blog/blog-is-a-toy

我玩博客还是博客玩我？

https://rercel.com/blog/blog-is-a-toy

**Comment by seeyousuperman** [comment]
十分喜欢这篇文章，句句戳我心。

**Comment by 7wate** [comment]
深有同感，不过群体存在多样性，有些人就是折腾『电子宠物』的，有些人就是喜欢漂流瓶社交的。总之开心就好 ~

**Comment by ghost** [comment]
深有共鸣！
